app/src/main/java/ch/epfl/sdp/blindly/profile/Profile5.kt

```diff
@@ -0,0 +1,47 @@
+package ch.epfl.sdp.blindly.profile
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import ch.epfl.sdp.blindly.R
+import com.google.android.material.chip.ChipGroup
+
+
+class Profile5 : AppCompatActivity() {
+
+    private val limitSelection = 3
```

Comment:
Might be a good idea to rename this constant using capital letters

Reply:
Corrected in the final commit.